app/configs/config.py

```diff
@@ -69,23 +69,23 @@ def url_file_tag(self) -> str:
 
     @computed_field
     def url_upload_greenroom(self) -> str:
-        return f'{self.api_url}/upload/gr'
+        return f'{self.url_portal}/fileops/gr'
 
     @computed_field
     def url_upload_core(self) -> str:
-        return f'{self.api_url}/upload/core'
+        return f'{self.url_portal}/fileops/core'
 
     @computed_field
     def url_status(self) -> str:
         return f'{self.api_url}/portal/v1/files/actions/tasks'
 
     @computed_field
     def url_download_greenroom(self) -> str:
-        return f'{self.api_url}/portal/download/gr'
+        return f'{self.url_portal}/portal/download/gr'
```

Comment:
I think these should be fileops as well?
```suggestion
        return f'{self.url_portal}/portal/fileops/gr'
```
Since fileops integrates download and upload?